cves/CVE-2011-1772.yml

```diff
@@ -45,7 +45,7 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: Multiple vulnerabilities allowed attackers to execute XSS attacks through input vectors involving an action or method name.
```

Comment:
More detail in this description would be nice. Remember, you are trying to explain this to someone who doesn't know software security. For example, you may want to explain XSS more.

Reply:
Fixed this in the most recent commit, it should make it easier to understand now.